cmd/nomos/status/client.go

```diff
@@ -354,6 +358,18 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and root-reconciler deployment
+// If root-reconciler exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context) bool {
+	v, _ := c.ConfigManagement.Version(ctx)
+	_, operatorDeployment := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, ACMOperatorDeployment, metav1.GetOptions{})
```

Comment:
`operatorDeployment` is actually the get error. Can you rename the variable to make it clear?

Reply:
renamed variable to be clearer